<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>title</class>
 <widget class="QMainWindow" name="title">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seuil</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Configuration Des Seuils</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="g1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <attribute name="title">
        <string>2G</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="cps">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="ccs">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QPushButton" name="vccs_c">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="vcps_c">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="uccs">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="vdcs_m">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="vccs_n">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="udcs">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="vcps_n">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="vcps_m">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="ucps">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="dcs">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="w3">
          <property name="text">
           <string>CSSR CS (Avg)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="w2">
          <property name="text">
           <string>DROP CS (Avg)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="w4">
          <property name="text">
           <string>CSSR PS (Avg) </string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="vdcs_n">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QPushButton" name="vdcs_c">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QPushButton" name="vccs_m">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="vdps_m">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="vdps_n">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="vdps_c">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dps">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="udps">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="w1">
          <property name="text">
           <string>DROP PS (Avg)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="g2">
       <attribute name="title">
        <string>3G</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="w1_2">
          <property name="text">
           <string>DROP PS (Avg)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dps_2">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="udps_2">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="vdps_n_2">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="vdps_m_2">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="vdps_c_2">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="w4_2">
          <property name="text">
           <string>CSSR PS (Avg) </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="cps_2">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="ucps_2">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="vcps_n_2">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="vcps_m_2">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="vcps_c_2">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="w2_2">
          <property name="text">
           <string>DROP CS (Avg)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="dcs_2">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="udcs_2">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="vdcs_n_2">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="vdcs_m_2">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="vdcs_c_2">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="w3_2">
          <property name="text">
           <string>CSSR CS (Avg)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="ccs_2">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="uccs_2">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="vccs_n_2">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="vccs_m_2">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="vccs_c_2">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="g3">
       <attribute name="title">
        <string>4G</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="w1_3">
          <property name="text">
           <string>DROP PS (Avg)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dps_3">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="udps_3">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="vdps_n_3">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="vdps_m_3">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="vdps_c_3">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="w4_3">
          <property name="text">
           <string>CSSR PS (Avg) </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="cps_3">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="ucps_3">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="vcps_n_3">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="vcps_m_3">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="vcps_c_3">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="w2_3">
          <property name="text">
           <string>DROP CS (Avg)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="dcs_3">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="udcs_3">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="vdcs_n_3">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="vdcs_m_3">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="vdcs_c_3">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="w3_3">
          <property name="text">
           <string>CSSR CS (Avg)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="ccs_3">
          <property name="prefix">
           <string>%   </string>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="uccs_3">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="vccs_n_3">
          <property name="text">
           <string>Mineur</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="vccs_m_3">
          <property name="text">
           <string>Majeur</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="vccs_c_3">
          <property name="text">
           <string>Critique</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
